### PR TITLE
Fix misfit test issues

### PR DESCRIPTION
**Pull request by valschneider**
I've been hitting some nasty issues with the misfit tests since the introduction of the rt-app buffer phase. Some (most?) of it is due to shady things I wrote in that test, so I only really have myself to blame...

I tested this by keeping tasks pinned in the last phase (IOW forcing a failure scenario), and the detected big CPU idle times were exactly what I expected, so AFAICT this works fine.

### CODE REVIEW

File: lisa/trace.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def __init__(self,
                  trace_path,
                  plat_info=None,
                  events=None,
-                 normalize_time=True,
+                 normalize_time=False,
```
Comment (douglas-raillard-arm) [inline lisa/trace.py]: Assuming we don't have traces with more than ~10 days of uptime that should be alright. After that, the nanosecond information will become noise.

Comment (valschneider) [reply lisa/trace.py]: The thing I'm trying to "fix" is somewhat orthogonal to this; the thing is I can easily re-load a `TestBundle` and print some of its internal data to figure out what happened. The issue is that the internal `Trace` is loaded with the default `normalize_time` (thus `True`) and that isn't easily hijackable.

So I would print some df, get normalized timestamps, and wouldn't be able to compare this with kernelshark. Other than the long uptime issue, AFAIR the main reason for normalization was for "pretty plots". In my experience this has been more of an annoyance (my bread and butter being kernelshark), so I'd like to change that.

Comment (douglas-raillard-arm) [reply lisa/trace.py]: We can change the default if we really don't expect anything more than 10 days uptime (and we could add a warning for non normalized time in that case), but it's pretty easy to get a trace with the parameter you want:
```
trace = test_bundle.get_trace(events=['foo', 'bar'], normalize_time=False)
```
`test_bundle.trace` is meant to be suited to test methods more than anything else, conflicting requirements can either be met with `get_trace()` or by introducing a new property (maybe a more friendly `test_bundle.debug_trace` that would parse all events, skip trace_window and provide non-normalized timestamps ?)

File: lisa/tests/scheduler/misfit.py [hunk 2]
```diff
@@ -128,29 +129,20 @@ def duration(self):
...
+        phase_df.set_index("Time", inplace=True)
+        phase_df.sort_index(inplace=True)
 
-            return sdf[(sdf.next_comm == name) & (sdf["__cpu"] == task_cpu)].index[0]
-
-        return max(
-            get_start_time(sdf, task, profile)
-            for task, profile in self.rtapp_profile.items()
-        )
+        return phase_df[phase_df.phase == 1].index[-1]
```
[the remark this answers is not in the document]
Comment (douglas-raillard-arm) [reply lisa/tests/scheduler/misfit.py]: EDIT: there sadly is no satisfactory solution for now, we would need named phases, which are not there yet

**Pull request merged**